client/src/featureform/resources.py

```diff
@@ -104,6 +104,29 @@ def serialize(self) -> bytes:
         return bytes(json.dumps(config), "utf-8")
 
 
+@typechecked
+@dataclass
+class AWSCredentials:
+    def __init__(self,
+                 aws_access_key_id: str = "",
+                 aws_secret_access_key: str = "",):
+        empty_strings = aws_access_key_id == "" or aws_secret_access_key == ""
+        assert not empty_strings, Exception(
```

Comment:
Are asserts typically used in non-testing situations?

Reply:
Modified it to exceptions.